### PR TITLE
Make the runtime of Frames public.

### PR DESCRIPTION
**Pull request by hasenbanck**
This PR extends the API for Frames and makes the length and the runtime public. I need access to it for my custom "Cycle" implementation where I can toggle if I want to repeat a SFX and if I want to include a pause for this (for example the sound of  foot steps).

Since english is not my mother tongue, "runtime" may not be the right name.

### CODE REVIEW

File: src/frames.rs [hunk 1]
```diff
@@ -85,6 +85,16 @@ impl<T> Frames<T> {
         self.rate as u32
     }
 
+    /// Number of frames
+    pub fn len(&self) -> usize {
+        self.samples.len()
+    }
```
Comment (Ralith) [inline src/frames.rs:91]: This should already be available via the `Deref` impl.

**Review by Ralith — CHANGES_REQUESTED** [review]
Thanks! Seems like a fine helper to have.

File: src/frames.rs (hunk 1 above)
Comment (hasenbanck) [reply src/frames.rs:91]: You're right. Wonder how I could have missed that.

**Comment by hasenbanck** [comment]
Removed the unecessary `len` implementation.

**Pull request merged**